etl/pypasar/omop/note.py

```diff
@@ -22,15 +21,125 @@ def execute(self):
+                                CDM_PER.person_id AS person_id,
+                                CAST(clindoc.postop_clindoc_created_datetime AS date) AS note_date, -- get date only
+                                clindoc.postop_clindoc_created_datetime AS note_datetime,
+                                clindoc.postop_clindoc_item_name AS note_title,
+
+                            -- Handle any null values in note text
+                            CASE
+                                WHEN clindoc.postop_clindoc_value_text IS NOT NULL 
+                                THEN clindoc.postop_clindoc_value_text
+                                ELSE '0'
+                            END AS note_text,
+
+                                CDM_VisitOcc.visit_occurrence_id AS visit_occurrence_id,
+                                clindoc.postop_clindoc_item_description AS note_source_value
+                            FROM filtered AS clindoc
+                            -- Join tables needed for person_id, visit_occurrence_id, visit_detail_id
+                            JOIN {omop_schema}.person AS CDM_PER
+                                ON clindoc.anon_case_no=CDM_PER.person_source_value
+                            JOIN sessionIDs AS CDM_VisitOcc
+                                ON clindoc.session_id=CDM_VisitOcc.session_id
+                        '''
+                     ))
+
+                # Read from stg__note and insert into CDM table note
+                connection.execute(
+                    text(f'''
+                        INSERT INTO {omop_schema}.note (
+                            note_id,
+                            person_id,
+                            note_date,
+                            note_datetime,
+                            note_type_concept_id,
+                            note_class_concept_id,
+                            note_title,
+                            note_text,
+                            encoding_concept_id,
+                            language_concept_id,
+                            provider_id,
+                            visit_occurrence_id,
+                            visit_detail_id,
+                            note_source_value,
+                            note_event_id,
+                            note_event_field_concept_id
+                        )
+                        SELECT
+                            note_id,
+                            person_id,
+                            note_date,
+                            note_datetime,
+                            32879 AS note_type_concept_id,
+                            0 AS note_class_concept_id,  -- Put 0 temporary as source_to_concept not included
+                            note_title,
+                            note_text, -- all value text is NULL
+                            32678 AS encoding_concept_id, -- clarification states data stored in database is UTF-8
```

Comment:
For this it makes sense to mark as 0 for all corresponding source values except for UTF-8 (32678)

From documentation https://ohdsi.github.io/CommonDataModel/cdm54.html#note
![image](https://github.com/user-attachments/assets/bb0fee89-a583-40c6-aa7b-29964119d94d)

Reply:
As discussed, not concrete since source column isnt available.